lib/actions.js

```diff
@@ -22,7 +22,9 @@ var colors = {
   UPDATE_FAILED: 'red',
   UPDATE_COMPLETE: 'green',
   ROLLBACK_IN_PROGRESS: 'red',
-  ROLLBACK_COMPLETE: 'red'
+  ROLLBACK_COMPLETE: 'red',
```

Comment:
Other statuses we should drag in:
- ROLLBACK_FAILED
- UPDATE_ROLLBACK_FAILED

I think that covers everything.